otsserver/stamper.py

```diff
@@ -314,26 +308,27 @@ def __do_bitcoin(self):
                 logging.error("Failed to get block")
                 return
 
-            # the following is an optimization, by pre computing the serialization of tx
-            # we avoid this step for every unconfirmed tx
-            serde_txs = []
+            # the following is an optimization, by pre computing the tx_id we rapidly check if our unconfirmed tx
+            # is in the block
+            block_txs_id = set()
             for tx in block.vtx:
-                serde_txs.append((tx, tx.serialize(params={'include_witness':False})))
+                block_txs_id.add(tx.GetTxid())
```

Comment:
You can made this more clear by using iterator notation:

    block_txids = set(tx.GetTxid() for tx in block.vtx)

Reply:
a lot more pythonic! Is it ok to keep the var name `block_txs_id`?

Reply:
`block_txids` sounds a lot more natural IMO.